website/docs/r/azure_event_hub_log_source.html.markdown

```diff
@@ -0,0 +1,54 @@
+---
+layout: "sumologic"
+page_title: "SumoLogic: sumologic_azure_event_hub_log_source"
+description: |-
+  Provides a Sumologic Azure Event Hub Log Source in GA.
+---
+
+# sumologic_azure_event_hub_log_source
+Provides a [Sumologic Azure Event Hub Log Source].
+
+__IMPORTANT:__ The Azure Event Hub credentials are stored in plain-text in the state. This is a potential security issue.
+
+## Example Usage
+```hcl
+resource "sumologic_azure_event_hub_log_source" "terraform_azure_event_hub_log_source" {
+  name          = "Azure Event Hub Log Source"
+  description   = "My description"
+  category      = "azure/eventhub"
+  content_type  = "AzureEventHubLog"
+  collector_id  = "${sumologic_collector.collector.id}"
+  authentication {
+    type = "AzureEventHubAuthentication"
+	  shared_access_policy_name = "%s"
+	  shared_access_policy_key = "%s"
+  }
+  path {
+    type = "AzureEventHubPath"
+	  namespace     = "%s"
+	  event_hub_name = "%s"
+	  consumer_group = "%s"
+    region = "%s"
```

Comment:
indentation is off on some lines

Reply:
This is strange. Looks good on my end
<img width="850" alt="Screenshot 2024-04-30 at 10 06 33 AM" src="https://github.com/SumoLogic/terraform-provider-sumologic/assets/124813412/3806783a-952e-4293-ac77-65c99b6cd796">